.vsts-ci.yml

```diff
@@ -49,6 +49,8 @@ jobs:
       gpu_source:
         TASK: gpu
         METHOD: source
+        # on Ubuntu 14.04, gpu_source build segfaults with Python 3.9.9
+        PYTHON_VERSION: 3.9.6
```

Comment:
Let's try to simplify with conda now. Maybe that issue was related only to mamba.
```suggestion
```

Reply:
fine with me. You can directly push any changes you want to this PR.

Reply:
Doesn't work. Reverted in 5e033adc726e8ceb34b35d731b019698045afd16.